api/maven-api-core/src/main/java/org/apache/maven/api/SourceRoot.java

```diff
@@ -37,11 +37,23 @@ public interface SourceRoot {
      * The path is relative to the <abbr>POM</abbr> file.
      *
      * <h4>Default implementation</h4>
-     * The default value is <code>src/{@linkplain #scope() scope}/{@linkplain #language() language}</code>
-     * as a relative path. Implementation classes may override this default with an absolute path instead.
+     * The default value depends on whether a {@linkplain #module() module name} is specified in this source root:
```

Comment:
```suggestion
     * The default value depends on whether a {@linkplain #module() module name} is specified in this source root:
```
default value -> `singular`